cmake/modules/jq.cmake

```diff
@@ -5,6 +5,9 @@ option(USE_BUNDLED_JQ "Enable building of the bundled jq" ${USE_BUNDLED_DEPS})
 
 if(JQ_INCLUDE)
 	# we already have jq
+	if(NOT TARGET jq)
```

Comment:
Eg: in this case, when `USE_BUNDLED_JQ` is false, we are not creating the target.

Reply:
Hmm I can see us not creating the target when `NOT USE_BUNDLED_JQ` but not otherwise. When `USE_BUNDLED_JQ`, we should get the target from `ExternalProject_Add(jq ...)`.

It's actually interesting that this builds now, probably because we `include(jq)` (and other modules?) twice (once in the top level CMakeLists.txt, once in cmake/modules/libsinsp.cmake).

Yeah, I'll move the target to the end of each module.

Reply:
> Yep! Perhaps i explained myself badly :D

No, I just reread your comment. Don't use no double negatives so early in the morning ;)